src/applications/claims-status/components/FilesOptional.jsx

```diff
@@ -7,7 +7,7 @@ function FilesOptional({ id, item }) {
   return (
     <va-alert
       class="optional-alert vads-u-margin-bottom--2"
-      status="default"
```

Comment:
updated since default is no longer used and its info instead [here](https://design.va.gov/storybook/?path=/docs/uswds-va-alert--default)